### ISSUE TITLE
Key mapping buttons missing

### ISSUE THREAD

**Issue by livinlicious** [jimgraygit/Onion#60]
I went through the installer process and everything worked.
In the menu everything works, but I noticed in the Games the DPAD didnt work.
So in the Menu I looked it up and it seems the DPAD, Start, Select cant be edited:
![download](https://user-images.githubusercontent.com/99683981/153926235-97d9304d-91c6-4e5b-857a-19b91a25fbac.png)

In the Test Hardware Menu everything works and is responsive. Also in the Emulator (Retroarch) it works.
But since its not mapped to the dpad, I guess it wont work?

I put in the former SD Card (the one that came with the device) and in it the same problem is now there.
Basically the device doesnt work anymore.

Any ideas how to proceed?

**Comment by Sichroteph** [comment]
Something is wrong with your configuration. Please make a fresh Onion install.

**Comment by livinlicious** [comment]
Hi,
can I request help in at least restoring the stock configuration somehow?
I tried inserting the fw and reflashing (like update process) but it only shows me the charging symbol and does nothing.

I would at least like to have the initial version running again. Maybe reflashing Onion might fix it as well.

**Comment by Sichroteph** [comment]
The input driver changed with the last onion version, I believe it will solve your issue.
